scripts/parse.py

```diff
@@ -18,12 +18,25 @@
 # along with taserver.  If not, see <http://www.gnu.org/licenses/>.
 #
 
-from typing import cast, Optional, List, Union, Set, Dict, Tuple, Generator, TextIO, BinaryIO
+from typing import TypeVar, cast, Optional, List, Union, Set, Dict, Tuple, Generator, TextIO, BinaryIO
 
 import csv
 import io
 import struct
-import sys
+import click
+
+K = TypeVar('K')
+V = TypeVar('V')
+
+
+def merge_value_dicts(ds: List[Dict[K, Set[V]]]) -> Dict[K, Set[V]]:
+    r = dict()
```

Comment:
Sure.

I'll leave the TypeVars because they're just generics (could be `KeyType` and `ValueType` I guess, but single letters is widely used enough convention IMO).

Reply:
Ok